inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -810,24 +811,25 @@ void MKLDNNGraph::Infer(MKLDNNInferRequest* request, int batch) {
 
     ENABLE_CPU_DEBUG_CAP(NodeDumper nd(config.debugCaps, infer_count));
 
-    for (int i = 0; i < graphNodes.size(); i++) {
+    for (auto &node : graphNodes) {
         if (request != nullptr) {
             request->ThrowIfCanceled();
         }
 
-        PERF(graphNodes[i]);
+        PERF(node);
```

Comment:
Was a bit discussed.
Since now we will have timestamps per single execution (node just average) I propose to start measuring right before **node->execute()** since the main goal is to measure just node execution time and not node execution + Infer() checks overhead. Even if this overhead could be interesting.